cmd/immutestapp/immutestapp.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019-2020 vChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	c "github.com/codenotary/immudb/cmd"
+	"github.com/spf13/cobra"
+)
+
+var App = "immutestapp"
+var Version = "v0.0.1"
```

Comment:
Why we need to populate the var version? It should arrives in the compiling time. Shouldn't?